source/com.google.accompanist/accompanist-drawablepainter/External-Dependency-Info.txt

```diff
@@ -0,0 +1,217 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do not translate or localize
+
+Xamarin Components for Tink Android
+third party material from the projects listed below. The original copyright 
+notice and the license under which Microsoft received such third party 
+material are set forth below.  Microsoft reserves all other rights not 
+expressly granted, whether by implication, estoppel or otherwise.
+
+########################################
+# Tink Android
```

Comment:
If these are all the same license, it makes more sense to put the file once in the `template` folder and include it from there.

Edit: Oh, I guess they are already part of templates, so all of these per-project License/TPN files seem extraneous.